kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/producer/KafkaPublisher.java

```diff
@@ -36,6 +36,7 @@
 import org.slf4j.LoggerFactory;
 
 import java.util.Objects;
+import java.util.Optional;
```

Comment:
Just thought about it, but by touching these files, the copyright notice's date should change to extend to this year.
You can have Intellij do this for you if you add the copyright template to your configuration.
You can find the template [here](https://github.com/AxonFramework/AxonFramework/blob/master/copyright-template.xml), by the way.

Reply:
So, this counts for all your Java files in this PR, actually :sweat_smile:

Reply:
Set the template, and fixed for all